### ISSUE TITLE
Regarding public and private test sets

### ISSUE THREAD

**Issue by iyamabhishek** [A2Zadeh/Social-IQ#6]
Hi @A2Zadeh,

Thank you for releasing the public test set. I had three questions regarding the test sets.

1. Many folders of the public test set are empty. Like the Segmented/Audio, Segmented/Frames, Segmented/Transcripts and Full/Audio. Is this okay or am I missing some files?
2. Will you release the features computed in .csd format for the public test set like the train and validation set?
3. Since the paper reports the final score on the private test set, what do you suggest is the best way to meaningfully compare my systems result with the score stated in the paper?

**Comment by iyamabhishek** [comment]
Hi @A2Zadeh,

I wanted to check if there's an update to this issue kindly.

**Comment by A2Zadeh** [comment]
@iyamabhishek sorry this fell through the cracks. Feel free to email me directly if I forgot to respond on an issue. 

1. Segmented ones are all extracted from segmented video. I will try to update this over this coming weekend. 

2. Yes, I will at some point in the next month. I am starting a few new models on Social-IQ and will do this also.

3. You can submit to the private test set. I would be happy to run that for you. Otherwise, you have to reimplement the existing models on the public test. I will actually do this for some of the upcoming work.
